priv/repo/migrations/20161128200526_create_user.exs

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.Repo.Migrations.CreateUser do
+  use Ecto.Migration
+
+  def change do
+    create table(:users) do
+      add :screen_name, :string, null: false
+      add :profile_image_url, :string
+      add :twitter_id, :integer, null: false
+      add :name, :string
```

Comment:
Name is the literal name i.e. "Scott Batson."  Screen name is your twitter handle "@ScottJBatson" and twitter_id is a unique identifier twitter uses as you can change your screen name or name at any time.